sample/src/main/java/com/clevertap/demo/MyApplication.kt

```diff
@@ -149,4 +152,19 @@ class MyApplication : MultiDexApplication(), CTPushNotificationListener, Activit
 
     override fun onActivityDestroyed(activity: Activity) {
     }
+
+    override fun onInboxButtonClick(payload: HashMap<String, String>?) {
+        Log.v("AppInbox", "InboxButtonClick with payload: $payload")
+        //dismissAppInbox()
+    }
+
+    override fun onInboxItemClicked(message: CTInboxMessage?, itemIndex: Int, buttonIndex: Int) {
+        Log.v("AppInbox", "InboxItemClicked at $itemIndex position with button-index: $buttonIndex")
+        //dismissAppInbox()
```

Comment:
add logic for button type check and then dismiss. For all buttons

Reply:
This is done.